source/contracts/reporting/Market.sol

```diff
@@ -63,17 +64,18 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
 
     function initialize(IReportingWindow _reportingWindow, uint256 _endTime, uint8 _numOutcomes, uint256 _numTicks, uint256 _feePerEthInAttoeth, ICash _cash, address _creator, address _designatedReporterAddress) public payable beforeInitialized returns (bool _success) {
         endInitialization();
-        require(address(_reportingWindow) != NULL_ADDRESS);
-        require(_numOutcomes >= 2);
-        require(_numOutcomes <= 8);
+        require(_numOutcomes >= 2 && _numOutcomes <= 8);
         require((_numTicks.isMultipleOf(_numOutcomes)));
         require(feePerEthInAttoeth <= MAX_FEE_PER_ETH_IN_ATTOETH);
         require(_creator != NULL_ADDRESS);
         require(_cash.getTypeName() == "Cash");
-        // FIXME: require market to be on a non-forking universe; repeat this check up the stack as well if necessary (e.g., in reporting window)
-        // CONSIDER: should we allow creator to send extra ETH, is there risk of variability in bond requirements?
-        require(msg.value == MarketFeeCalculator(controller.lookup("MarketFeeCalculator")).getMarketCreationCost(_reportingWindow));
         reportingWindow = _reportingWindow;
+        require(address(getForkingMarket()) == NULL_ADDRESS);
+        // CONSIDER: should we allow creator to send extra ETH, is there risk of variability in bond requirements?
```

Comment:
I believe the answer to this CONSIDER is now "yes".  Depending on when the transaction is mined relative to when it was created, the fees may change so we should support the creator sending excess ETH and refunding unused at the end of the transaction.